about.html

```diff
@@ -12,55 +13,87 @@
 </head>
 
 <body>
+ <img class="myheader" src="img/lambda-black.png" alt="lambda header">
+ 
     <div class="container about-page">
-     
-    About
+     <nav class="menu">
+      
+      
+<a href="index.html"> Home</a>
+<a href="#"> About </a>
+<a href="#"> Products</a>
+<a href="#"> Blog </a>
+<a href="#"> Contact</a>
+</nav>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
+<img class="img1" src="img/jumbo-about.png" alt="city photo">
```

Comment:
Very close on this. We'll talk more, but you either need to put the`img` in the `nav` tags or create another division, maybe a `header` tag, that could house all those elements. We want them to be responsive together.